.github/workflows/ci.yml

```
@@ -40,3 +41,11 @@ jobs:
         run: |
           docker login -u "${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}" --password-stdin <<<"${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}" quay.io
           make push
+      - name: Re-tag and Push Latest
+        run: |
+          if [[ "${{ github.ref }}" =~ refs/tags/* ]]; then
```

Sorry, I just realized this should be
```suggestion
          if [[ "${{ github.ref }}" != refs/tags/* ]]; then
```